packages/changed-elements-react/src/widgets/ChangedElementsWidget.tsx

```diff
@@ -59,7 +58,7 @@ export interface ChangedElementsWidgetProps {
 */
   onJobUpdate?: (comparisonJobUpdateType: ComparisonJobUpdateType, jobAndNamedVersions?: JobAndNamedVersions) => Promise<void>;
   /** Optional prop for a user supplied component to handle managing named versions.*/
-  manageNamedVersionProps?: ComponentProps<typeof ManageNamedVersions>;
+  manageNamedVersionsSlot: ReactNode | undefined;
```

Comment:
The property should have remained optional.

`manageNamedVersionsSlot?: ReactNode | undefined`